docs/README.md

```diff
@@ -65,7 +65,7 @@ new Vue({
 });
 ```
 
-From now on, you can use all [Vue InstantSearch components](using-components.md) in your templates throughout the whole application.
```

Comment:
Used components to shorten and also because components is split between "registering" and "using"